client/my-sites/themes/theme-showcase.scss

```
@@ -125,9 +129,96 @@
 			}
 		}
 	}
+
+	.theme__search {
+
+		@include break-mobile {
+			width: 100%;
+		}
+
+		@include break-small {
+			width: 75%;
+		}
+
+		@include break-large {
+			width: 50%;
+		}
+
+		.section-nav-tabs__dropdown {
+			box-shadow:
+				0 0 0 0 rgba(38, 19, 19, 0.03),
+				0 1px 2px 0 rgba(38, 19, 19, 0.03),
+				0 4px 4px 0 rgba(38, 19, 19, 0.03),
+				0 9px 5px 0 rgba(38, 19, 19, 0.02),
+				0 16px 6px 0 rgba(38, 19, 19, 0),
+				0 25px 7px 0 rgba(38, 19, 19, 0);
+
+			.select-dropdown__header {
+				border-width: 0;
+				color: #575d63;
+				font-size: 1rem;
+				line-height: 1.5;
+				font-weight: 400;
+			}
+		}
+	}
 }
 
 .theme-showcase {
+
+	.theme__search {
+		.keyed-suggestions {
+			top: 42px;
+		}
+
+		.select-dropdown__options {
+			position: absolute;
+			width: 100%;
+
+			@include break-mobile {
+				position: inherit;
+			}
+		}
+
+		@include break-mobile {
+			flex-direction: row;
+		}
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+		align-items: center;
+		padding-bottom: 10px;
+
+		.search-themes-card {
+			.search {
+				height: 43px;
+			}
+		}
+
+		.section-nav-tabs__dropdown .select-dropdown__container {
+			width: 100%;
+		}
+
+		.theme__search--input {
+			flex: 1;
+			width: 100%;
+			padding: 0 0 10px 0;
+			@include break-mobile {
+				flex: 2;
+				padding: 0 10px 0 0;
+			}
+		}
+
+		.section-nav-tabs__dropdown {
+			flex: 1;
+			width: 100%;
```

I think a fixed width works better here
```suggestion
			width: 250px;
```

---

Depending on the break point we would need to add width 100%, in order to reduce the amount of break point I'll keep it as it is.